appletree/context.py

```diff
@@ -390,7 +391,17 @@ def update_parameter_config(self, likelihoods):
                 self.par_config.pop(p)
         return needed_parameters
 
-    def lineage(self, data_name: str = "cs2"):
-        """Return lineage of plugins."""
-        assert isinstance(data_name, str)
-        pass
+    @property
+    def lineage_hash(self):
```

Comment:
Can we also add the backend filename or something equivalent to the context hash? Sometimes the context is initialized from a backend h5 file which does not have `self.instruct` etc.

Reply:
Anyways, let's do it if necessary in the future.